p2p/wireformat.go

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright (C) 2020 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package p2p
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"io"
+	"net/textproto"
+	"strconv"
+
+	"github.com/rs/zerolog/log"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/mysteriumnetwork/node/p2p/compat"
+	"github.com/mysteriumnetwork/node/pb"
+)
+
+const maxTransportMsgLen = 128 * 1024
+
+func init() {
+	// This is needed to initialise global common headers map state internally
```

Comment:
Two of my spellcheckers suggest to use `z` 😂 :
```suggestion
	// This is needed to initialize global common headers map state internally
```